Makefile

```diff
@@ -4,7 +4,7 @@ GOVET=$(GOCMD) vet
 BINARY_NAME=local-ai
 
 # llama.cpp versions
-GOLLAMA_VERSION?=50cee7712066d9e38306eccadcfbb44ea87df4b7
+GOLLAMA_VERSION?=f03869d188b72c8a617bea3a36cf8eb43f73445c
```

Comment:
this PR might need to wait for me testing on GPU if you don't have one to test it out. 

Things tends to be pretty delicate in that domain when pulling changes from upstream (llama.cpp)